python_modules/dagster/dagster/core/execution/stats.py

```diff
@@ -163,6 +165,9 @@ def build_run_step_stats_from_events(
             step_attempts.append(
                 RunStepMarker(start_time=attempt_start, end_time=step_stats["end_time"])
             )
+        else:
+            step_attempts.append(RunStepMarker(start_time=attempt_start))
```

Comment:
this is, in essence, adding step stats for steps that have not completed.  I think this will have implications (break things) in the `verify_step` call in `dagster/cli/api.py`

Reply:
True. Going to remove this line, we'll rely on `StepEventStatus` to determine in-progress steps